src/test/datascience/notebook/intellisense/semanticTokens.vscode.test.ts

```diff
@@ -126,18 +126,13 @@ suite('DataScience - VSCode semantic token tests', function () {
         await insertCodeCell(
             'import sqllite3 as sql\n\nconn = sql.connect("test.db")\ncur = conn.cursor()\n# BLAH BLAH'
         );
-        await insertCodeCell('\n');
+        await insertCodeCell(
```

Comment:
Not sure why it was building the document like this, but it didn't seem relevant to the test and it was having issues on CI (second cell wasn't populated).

Reply:
Not sure why the cell wasn't being added, but I believe this test was verifying editing cells gets semantic tokens.